api-call.py

```diff
@@ -17,13 +38,53 @@ def format_result(data):
     print(text)
 
 
+def show_tasks():
+    print("Which API data you want to check \n")
+    print("1. Beavers Bus")
+    print("2. Terms")
+    print("3. Quit")
+
+def get_user_choice():
+    choice = input("Enter your choice (1 or 2 or 3): ")
+    return choice
+
+def get_url(choice):
+    if choice == "1":
+        url = "https://api.oregonstate.edu/v1/beaverbus/routes"
+        return url
+    elif choice == "2":
+        url = "https://api.oregonstate.edu/v1/terms"
+        return url
+    elif choice == "3":
+            print("Exisiting...")
+    else:
+        print("Invalid choice. Please try again.")
+
 
 
 if __name__ == "__main__":
-    parameters = {
+
+    access_token = get_access_token()
+
+    access_token = "Bearer " + access_token
+
+    header={"Content-Type": "application/x-www-form-urlencoded;charset=utf-8" , "Authorization":access_token}
+
+
+    while True:
+        parameters = {
             
         }
 
-    header={"Content-Type": "application/x-www-form-urlencoded;charset=utf-8" , "Authorization":"Bearer CUTOuxWmuRfKQKynBGjFGmBROM9R"}
-        
-    api_call =  get_api_data("https://api.oregonstate.edu/v1/beaverbus/routes", parameters, header)
+        show_tasks()
+
+        user_choice = get_user_choice()
+
+        api_url = get_url(user_choice)
+
+        if user_choice == "3":
+            break
+
+        api_call =  get_api_data(api_url, parameters, header)
+
+            
```

Comment:
We follow our coding guidance for every language (Python - [pep8](https://peps.python.org/pep-0008/)/[flake8](https://flake8.pycqa.org/en/latest/) bases). Please take a look at [here](https://github.com/osu-mist/style-guides/blob/master/Python.md) and make sure your code also adheres to our coding style.

Reply:
improved this part.